client/src/pages/UserProfile.js

```diff
@@ -0,0 +1,234 @@
+// Forewarning, I have no idea what I'm doing when it comes to React
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Button from "@material-ui/core/Button";
+import {
+  Avatar,
+  Box,
+  Grid,
+  List,
+  ListItem,
+  ListItemText,
+} from "@material-ui/core";
+import Typography from "@material-ui/core/Typography";
+import profile_pic from "../assets/images/profile_pic.png";
+import map from "../assets/images/map.png";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  outerGrid: {
+    display: "flex",
+    justifyContent: "center",
+    background: "#F8F8FF",
+  },
+  innerGrid: {
+    boxShadow: theme.shadows[5],
+    background: "#F8F8FF",
+  },
+  // Header
+  cardHeader: {
+    background: "white",
+    width: "100%",
+    display: "flex",
+    direction: "row",
+    alignSelf: "center",
+  },
+  // Left header
+  cardHeadLeft: {
+    marginTop: "10%",
+  },
+  headerLeftContainer: {
+    border: "1px solid lightGrey",
+    paddingBottom: "3.5em",
+    ["@media (max-width:800px)"]: { paddingBottom: "3em" },
+  },
+  cardHeadAvatar: {
+    alignSelf: "center",
+    border: "3px solid white",
+    boxShadow: theme.shadows[5],
+    marginBottom: "1em",
+    width: "6em",
+    height: "6em",
+  },
+  cardHeadLocation: {
+    paddingBottom: "1.5em",
+    color: "grey",
+  },
+  cardSendMessage: {
+    border: "2px solid #FF510C",
+    color: "#FF510C",
+    borderRadius: "0",
+    paddingTop: "1em",
+    paddingBottom: "1em",
+    paddingLeft: "2.5em",
+    paddingRight: "2.5em",
+    alignSelf: "center",
+  },
+
+  // Right header
+  cardHeadRight: {
+    height: "100%",
+    display: "flex",
+    justifyContent: "center",
+  },
+  headRightContainer: {
+    display: "flex",
+    flexWrap: "wrap",
+    // border: "2px solid red"
+  },
+  bioContainer: {
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "center",
+  },
+  cardBio: {
+    //  border: "3px solid black",
+    display: "flex-column",
+    alignSelf: "center",
+    paddingBottom: "2em",
+    ["@media (max-width:800px)"]: { marginTop: "2em" },
+  },
+  bioContainer: {
+    display: "flex-column",
+    alignSelf: "center  ",
+  },
+  bioHead: {
+    color: "grey",
+    paddingBottom: "1em",
+  },
+  biography: {
+    color: "grey",
+    marginBottom: "0.75em",
+  },
+  cardFavorites: {
+    // border: "3px solid red",
+    alignSelf: "center",
+    display: "flex",
+    flexDirection: "column",
+    justify: "space-between",
+  },
+  favoritesList: {
+    // border: "3px solid black",
+    display: "flex",
+    flexDirection: "row",
+    ["@media (max-width:600px)"]: {
+      flexDirection: "column",
+      marginBottom: "1em",
+    },
+    padding: 0,
+  },
+  favoriteCuisine: {
+    height: "2em",
+    display: "flex",
```

Comment:
when using Grids you shouldn't need separate "flex".
Grids already include flex built in. Look at the [Grid API](https://material-ui.com/api/grid/) to get a sense of what you can do with it.